enterprise/views.py

```diff
@@ -1022,6 +1032,7 @@ def get_enterprise_course_enrollment_page(
             'continue_link_text': _('Continue'),
             'level_text': _('Level'),
             'effort_text': _('Effort'),
+            'duration_text': _('Duration'),
```

Comment:
suggestion: `duration_title`

Reply:
Following the same pattern as for effort_text